tests/common/test_dataset_generators.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from torch.random import manual_seed
+from transformers import AutoModelForCausalLM
+from transformers import AutoTokenizer
+
+import nncf
+from nncf.data import generate_text_data
+from tests.cross_fw.shared.helpers import load_json
+from tests.cross_fw.shared.paths import TEST_ROOT
+
+manual_seed(0)
```

Comment:
Please use fixture to set seed like https://github.com/openvinotoolkit/nncf/blob/develop/tests/torch/conftest.py#L256
to guaranty that it will not affect to another tests and seed will be set individually for each test

Reply:
Done.